<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cross_correlationDialog</class>
 <widget class="QDialog" name="cross_correlationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cross-correlation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QCheckBox" name="orderCheckBox">
     <property name="text">
      <string>Order by match</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="OscillogramWidget" name="oscillogramWidget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(11, 64, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SpectrogramWidget" name="spectrogramWidget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(49, 8, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="matchTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Match</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Offset</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OscillogramWidget</class>
   <extends>QWidget</extends>
   <header>duetto.widgets.OscillogramWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectrogramWidget</class>
   <extends>QWidget</extends>
   <header>duetto.widgets.SpectrogramWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
